posthog/api/person.py

```diff
@@ -257,7 +257,7 @@ def funnel(self, request: request.Request, **kwargs) -> response.Response:
         if not results_package:
```

Comment:
Is this new pattern not applicable to the regular person list?

Reply:
.. What do you mean? 🤔

Reply:
Oh it only happens in one place but posthog/api/cohort.py line 217 we're still using the postgres result and not paginating based on clickhouse raw count result

Reply:
Nice catch, have deleted the `should_paginate` function and replaced all usage